src/code_8048480.c

```diff
@@ -549,7 +549,7 @@ void CheriBerryItemAction(Entity *pokemon, Entity *target)
 
 void PechaBerryItemAction(Entity *pokemon, Entity *target)
 {
-    if((u8)(target->axObj.info->nonVolatile.nonVolatileStatus - 2) <= 1)
+    if((u8)(GetEntInfo(target)->nonVolatile.nonVolatileStatus - 2) <= 1)
```

Comment:
I know, that's not related, but since you're at it, maybe you could fix that as well? It's prolly `nonVolatileStatus == 1 || nonVolatileStatus  == 2` or sth similar